test/ts-types/main.ts

```diff
@@ -0,0 +1,27 @@
+import {getDocument} from "pdfjs-dist";
+
+class MainTest {
+    task: ReturnType<typeof getDocument> | undefined
```

Comment:
Also use two-space indentation in this file for consistency with the other code.